templates/pastes.html

```diff
@@ -0,0 +1,74 @@
+{% extends 'base.html' %}
+{% block head %}
+  {% load static %}
+{% endblock %}
+{% block content %}
+  <section class="page">
+    <header>
+      <figure>
+        <img src="{% static "/assets/images/gridpaste-logo.svg" %}" alt="logo" />
+      </figure>
+    </header>
+    <div id="content">
+        <a href="/">
+          <div class="button new">New</div>
+        </a>
+        <ol id="items">
+            {% for paste in pastes %}
+                <li data-id="{{paste.id|escape}}">
```

Comment:
Why are you explicitly escaping all of these values?

Did you turn off auto-escaping somewhere?